DESCRIPTION

```diff
@@ -1,7 +1,15 @@
 Package: genefilter
 Title: genefilter: methods for filtering genes from high-throughput experiments
 Version: 1.81.0
-Author: Robert Gentleman, Vincent J. Carey, Wolfgang Huber, Florian Hahne
+Authors@R: c(
+    person("Robert", "entleman", role = "aut"),
+    person("Vincent J.", "Carey", role = "aut"),
+     person("Wolfgang", "Huber", role = "aut"),
+      person("Florian", "Hahne", role = "aut"),
+    person("Khadijah", "Amusat", role = "ctb",
+        comment = "Converted genefilter vignette from Sweave to RMarkdown / HTML."),
+    person("Bioconductor Package Maintainer",
+        email = "maintainer@bioconductor.org", role = "cre"))
 Description: Some basic functions for filtering genes.
 Maintainer: Bioconductor Package Maintainer
         <maintainer@bioconductor.org>
```

Comment:
Remove the lines

```
Maintainer: Bioconductor Package Maintainer
        <maintainer@bioconductor.org>
```
This is because the maintainer is now in `Authors@R`.